assignment/week03/travel-recommendation/src/components/NonRandom/NonRandom.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect, useState } from 'react';
+import { BtnBox, MainBox,Description,AnswerBox, Btn, LevelBox } from '../../styles/Design';
+import { questionList, temporaryAnswer } from './QuestionList';
+
+const NonRandom = ({id, setId, setIsStarted, calculator, setCalculator}) => {
+    const [nextBtnContent, setNextBtnContent]=useState('다음으로');
+    const [loaded,isLoaded]=useState(false);
+
+    useEffect(()=>{
+        id>=2 ? setNextBtnContent("결과보기") : setNextBtnContent("다음으로");
+        isLoaded(true);
+    },[id]);
+
+    const onAnswerClick=(idx)=>{
+        const newTemporary = temporaryAnswer[id].map((each,i)=>idx===i ? true : false);
```

Comment:
```suggestion
        const newTemporary = temporaryAnswer[id].map((each,i)=>return idx === i);
```
-> 요거는 이런식으루 해도 될것 같당!